src/WebJobs.Script/Rpc/Configuration/GenericWorkerProvider.cs

```diff
@@ -15,11 +15,13 @@ internal class GenericWorkerProvider : IWorkerProvider
     {
         private WorkerDescription workerDescription;
         private List<string> arguments;
+        private string pathToWorkerDir;
```

Comment:
All of the private members here are missing the `_` prefix. Is StyleCop disabled for this type?

Reply:
I didn't see any errors/warnings. I didn't do anything to disable this type.

Reply:
Will look at it... can we just update to follow the standards on the project?